lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -34,6 +35,10 @@ define((require, exports, module) ->
     }
 
     initKaTeX = (onLoaded) ->
+      # Loading banned sequences file
+      response = $.getJSON(require.toUrl("./banned_token_sequences.json"))
```

Comment:
Maybe create a module and add dependency into katex? In production setup katex and this list will be compiled into a single file which will be loaded as a whole